### PR TITLE
gogen

### PR DESCRIPTION
**Pull request by chenyanchen**
env:
  go version go1.14.4 darwin/amd64
  protoc-gen-go v1.25.0
  protoc v3.11.0
cmd:
```bash
$ go get -u github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway
$ go get -u github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger
$ go get -u github.com/golang/protobuf/protoc-gen-go
$ make gogen
$ go mod tidy
```

### CODE REVIEW

File: golang/iotexapi/api.pb.go [hunk 1]
```diff
@@ -1,217 +1,307 @@
+// Copyright (c) 2019 IoTeX
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+// To compile the proto, run:
+//      protoc -I. -I ./../types --go_out=plugins=grpc:$GOPATH/src *.proto
+
```
Comment (dustinxie) [inline golang/iotexapi/api.pb.go:9]: if you manually add these, remove it
see below

File: go.mod [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
-	golang.org/x/net v0.0.0-20190503192946-f4e77d36d62c // indirect
-	golang.org/x/sys v0.0.0-20190502175342-a43fa875dd82 // indirect
-	golang.org/x/text v0.3.2 // indirect
-	google.golang.org/genproto v0.0.0-20190502173448-54afdca5d873 // indirect
-	google.golang.org/grpc v1.20.1
+	github.com/allegro/bigcache v1.2.1 // indirect
+	github.com/golang/protobuf v1.4.2
+	github.com/grpc-ecosystem/grpc-gateway v1.14.6
+	github.com/iotexproject/go-pkgs v0.1.1
+	github.com/iotexproject/iotex-antenna-go v1.0.0
```
Comment (dustinxie) [inline go.mod:8]: why this dependency? think it's only used in example

File: golang/iotexapi/api.pb.go [hunk 3]
```diff
@@ -1,217 +1,307 @@
...
+)
 
-// This is a compile-time assertion to ensure that this generated file
-// is compatible with the proto package it is being compiled against.
-// A compilation error at this line likely means your copy of the
-// proto package needs to be updated.
-const _ = proto.ProtoPackageIsVersion3 // please upgrade the proto package
+// This is a compile-time assertion that a sufficiently up-to-date version
+// of the legacy proto package is being used.
+const _ = proto.ProtoPackageIsVersion4
```
Comment (dustinxie) [inline golang/iotexapi/api.pb.go:41]: why it becomes v4

File: golang/iotexapi/api.pb.go (hunk 1 above)
Comment (chenyanchen) [reply golang/iotexapi/api.pb.go:9]: no, it's generate from source *.proto files

File: golang/iotexapi/api.pb.go (hunk 3 above)
Comment (chenyanchen) [reply golang/iotexapi/api.pb.go:41]: maybe all unexpected changes cause ProtoPackageIsVersion4, but I dont known why, I'll check it later

Comment (chenyanchen) [reply golang/iotexapi/api.pb.go:41]: resolved
cause `go get -u github.com/golang/protobuf/protoc-gen-go` upgrade protoc-gen-go => 1.4.2(current latest version)
exec `go get github.com/golang/protobuf/protoc-gen-go@v1.3.1` to get  v1.3.1

Comment (dustinxie) [reply golang/iotexapi/api.pb.go:41]: can you regenerate using v1.3.1? 
at this point we don't want to upgrade to V4 now, that can be planned in near future maybe

**Pull request closed without merging**